src/main/java/de/tum/cit/aet/artemis/programming/domain/AuthenticationMechanism.java

```diff
@@ -25,4 +25,8 @@ public enum AuthenticationMechanism {
      * The user attempted to authenticate to the LocalVC using either a user token or a participation token
      */
     VCS_ACCESS_TOKEN,
+    /**
+     * The authentication header was missing
+     */
+    NONE
```

Comment:
I think `NONE` is kind of non-descriptive. `NO_HEADER` or smth similar could work wonders to make it more intuitive.

Reply:
Good idea, I refactored it to  AUTH_HEADER_MISSING